### ISSUE TITLE
The formatting command doesn't work with Node 12

### ISSUE THREAD

**Issue by lazarljubenovic** [angular/angular#32540]
# 🐞 bug report

### Affected Package

 The issue is caused by **the dev pipeline**.

### Is this a regression?

Kinda, because it used to work with Node 11? But not Angular's regression.

### Description

 A clear and concise description of the problem...

Using Node 12 and trying to run the formatting command breaks.

## 🔬 Minimal Reproduction

1. Clone the repo.
2. Use Node 12 (eg. `nvm use 12`).
3. Run `yarn gulp format`.
4. Cry.
5. Use Node 11 (eg. `nvm use 11`).
6. Run `yarn gulp format`.
7. Smile.

## 🔥 Exception or Error

With Node 12:

```
lazar@lazar:~/angular$ yarn gulp format:all
yarn run v1.17.3
$ /home/lazar/angular/node_modules/.bin/gulp format:all
fs.js:27
const { Math, Object } = primordials;
                         ^

ReferenceError: primordials is not defined
    at fs.js:27:26
    at req_ (/home/lazar/angular/node_modules/natives/index.js:143:24)
    at Object.req [as require] (/home/lazar/angular/node_modules/natives/index.js:55:10)
    at Object.<anonymous> (/home/lazar/angular/node_modules/vinyl-fs/node_modules/graceful-fs/fs.js:1:37)
    at Module._compile (internal/modules/cjs/loader.js:936:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10)
    at Module.load (internal/modules/cjs/loader.js:790:32)
    at Function.Module._load (internal/modules/cjs/loader.js:703:12)
    at Module.require (internal/modules/cjs/loader.js:830:19)
    at require (internal/modules/cjs/helpers.js:68:18)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

With Node 11:

```
lazar@lazar:~/angular$ yarn gulp format
yarn run v1.17.3
$ /home/lazar/angular/node_modules/.bin/gulp format
[19:59:46] Using gulpfile ~/angular/gulpfile.js
[19:59:46] Starting 'format:untracked'...
[19:59:46] Starting 'format:diff'...
[19:59:46] Finished 'format:untracked' after 416 ms
[19:59:46] Finished 'format:diff' after 379 ms
[19:59:46] Starting 'format:changed'...
[19:59:46] Finished 'format:changed' after 20 μs
[19:59:46] Starting 'format'...
[19:59:46] Finished 'format' after 3.84 μs
Done in 0.95s.
```

## 🌍  Your Environment

**Angular Version:**
```
"name": "angular-srcs",
"version": "9.0.0-next.5",
```

**Anything else relevant?**

Note that `package.json` claims that `12` is fine to use: 

https://github.com/angular/angular/blob/df5924abd0c4efe87e2fd189739e565346276b91/package.json#L10-L13

**Comment by lazarljubenovic** [comment]
Related: https://stackoverflow.com/questions/55921442/how-to-fix-referenceerror-primordials-is-not-defined-in-node

**Comment by pumano** [comment]
@lazarljubenovic looks like it's not related to angular itself. Also your library may not working with node12.

**Comment by lazarljubenovic** [comment]
My library? What library?

While it's not directly issue with Angular, it _is_ an issue with a command available on the repo which claims to work in Node 11 and 12, but doesn't. It can be solved by internally changing the Gulp version or by getting rid of gulp altogether, hence I believe it belongs to the issue tracker.

**Comment by devversion** [comment]
This has been fixed with #33348
